workbase/wrappers-bazel/npm_run_build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export PATH=`pwd`/external/nodejs/bin:$PATH
+export NODE_PATH=`pwd`/workbase/node_modules/
+
+pushd workbase
```

Comment:
What happens when you run `bazel build` not from the root workspace directory? will this script still work?

And can we have the filenames use hyphens instead of underscores?

Reply:
@haikalpribadi 
>What happens when you run bazel build not from the root workspace directory? will this script still work?

Yes, because `bazel` replicates workspace structure in sandbox

>And can we have the filenames use hyphens instead of underscores?

 Yup